src/index.html

```diff
@@ -3,11 +3,393 @@
+            </li>
+          </ul>
+        </nav>
+      </section>
+
+      <!-- middle -->
+      <input
+        type="checkbox"
+        class="header__menu-opener"
+        id="menuOpener"
+      >
+
+      <section class="menu">
+        <label for="menuOpener" class="menu__closer"></label>
+        <a href="#" class="big-logo menu__logo">
+          <img
+            src="images/logo.png"
+            alt="miami logo"
+            class="big-logo__img"
+          >
+        </a>
+        <div class="menu__big-link-container">
+          <label for="menuOpener" class="menu__link-closer">
+            <a href="#home" class="header__item">home</a>
+          </label>
+          <label for="menuOpener" class="menu__link-closer">
+            <a href="#aboutUs" class="header__item">about us</a>
+          </label>
+          <label for="menuOpener" class="menu__link-closer">
+            <a href="#gallery" class="header__item">gallery</a>
+          </label>
+          <label for="menuOpener" class="menu__link-closer">
+            <a href="#consultation" class="header__item">consultation</a>
+          </label>
+          <label for="menuOpener" class="menu__link-closer">
+            <a href="#testimonials" class="header__item">testimonials</a>
+          </label>
+          <label for="menuOpener" class="menu__link-closer">
+            <a href="#contacts" class="header__item">contacts</a>
+          </label>
+        </div>
+        <div class="menu__small-link-container">
+          <a href="tel:+12345555555" class="header__item header__item--menu-small-link">
+            +1 234 555-55-55
+          </a>
+          <a href="#" class="header__item header__item--menu-small-link header__item--small-underlined">
+            call to order
+          </a>
+        </div>
+      </section>
+      <!-- middle -->
+    </header>
+
+    <main class="content-container main-tag">
+      <article class="about-us" id="aboutUs">
+        <h2 class="about-us__top-title">
+          Miami best<br>Real estate <span class="bright-emphasis">here</span>
+        </h2>
+        <div class="about-us__text-box">
+          <h2 class="about-us__middle-title bright-emphasis">
+            Over 50 years experience in love.
+          </h2>
+          <p class="about-us__text">
+            Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical. <br><br>
+            The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.
+            <br>Working directly with developers - so have more useful information about the profitable deals.
+            <br><br>
+            <big>We will never be late!<br>Guaranteed secure transaction - 100%</big>
```

Comment:
remove extra <br> and wrap blocks in different `p` containers

Reply:
ok

Reply:
I meant you don't need so many `br`
those are different blocks, so create different `p` blocks for them

Reply:
no, i think it is more useful to use br

Reply:
you need to be able to change the gap between them, and this should be done with css

semantically, those are too separate blocks of text

Reply:
No. Maybe marketer will want to change text. May be he will want only one text-row. Maybe 3 rows. Different blocks is excessive